### PR TITLE
Review unit tests

### PR DESCRIPTION
**Pull request by bobtaylor12**
Added low level unit tests to improve code coverage

### CODE REVIEW

File: src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        //Test that valid parameters are handled correctly
+        ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
+        assertTrue(PopulationParameters.get().isValid());
+    }
+
+    @Test (expected = AssertionError.class)
+    public void testIsInValid() throws IOException {
+        //Test that invalid parameters are handled correctly
+        ParameterReader.readParametersFromFile("src/test/resources/test_invalid_params.json");
+        assertTrue(PopulationParameters.get().isValid());
```
Comment (johnnonweiler) [inline src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java:29]: Why not use `assertFalse()` here?

File: src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Test (expected = InvalidParametersException.class)
+    public void testNullParameters() {
+        //Test that null parameters are handled correctly
+        assertTrue(PopulationParameters.get().isValid());
```
Comment (johnnonweiler) [inline src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java:35]: I suggest removing the `assertTrue()` here too:
```suggestion
        PopulationParameters.get().isValid();
```

File: src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java (hunk 1 above)
Comment (bobtaylor12) [reply src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java:29]: ok

File: src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java (hunk 2 above)
Comment (bobtaylor12) [reply src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java:35]: ok

**Pull request merged**